config/splat.us.stno4.yaml

```diff
@@ -43,6 +43,8 @@ segments:
       - [0xCF4, data]
       - [0xD5C, .data, st_debug]
       - [0xDDC, data]
+      - [0x1880, .data, e_secrets]
+      - [0x18B0, data, e_secrets]
```

Comment:
`.data` marks this section as imported into C code. This is for `D_us_80181880`

`data` section (without the .) is not yet imported and relates to `D_us_801818B0` which is used by `func_us_801C89F4` in the same `e_secrets` file